validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -243,6 +263,8 @@ func (c *grpcValidatorClient) StartEventStream(ctx context.Context, topics []str
 }
 
 func (c *grpcValidatorClient) EventStreamIsRunning() bool {
+	c.eventStreamLock.Lock()
+	defer c.eventStreamLock.Unlock()
```

Comment:
Is this called from multiple go routines? If so, we may want to use RLock here

Reply:
No, starting should be done from only one place.